### ISSUE TITLE
Adventure Guide broken (retail)

### ISSUE THREAD

**Issue by jaredbwasserman** [jaredbwasserman/drift#51]
Dragging Adventure Guide causes strange behavior.

![WoWScrnShot_020123_224819](https://user-images.githubusercontent.com/5832654/216252362-7858fc77-4e99-434f-9dff-0608849b1a27.jpg)

**Comment by jaredbwasserman** [comment]
```
5x ...zzard_EncounterJournal/Blizzard_EncounterJournal.lua:3340: Action[SetPoint] failed because[SetPoint would result in anchor family connection]: attempted from: EncounterJournalTooltip:SetPoint.
[string "=[C]"]: in function `SetPoint'
[string "@Blizzard_EncounterJournal/Blizzard_EncounterJournal.lua"]:3340: in function <...zzard_EncounterJournal/Blizzard_EncounterJournal.lua:3336>

Locals:
(*temporary) = EncounterJournalTooltip {
 SetBorderBlendMode = <function> defined @SharedXML/SharedTooltipTemplates.lua:203
 NineSlice = Frame {
 }
 SetBackdropBorderColor = <function> defined @SharedXML/SharedTooltipTemplates.lua:195
 TooltipBackdropOnLoad = <function> defined @SharedXML/SharedTooltipTemplates.lua:172
 Item1 = EncounterJournalTooltipItem1 {
 }
 Item2 = EncounterJournalTooltipItem2 {
 }
 GetBackdropColor = <function> defined @SharedXML/SharedTooltipTemplates.lua:191
 headerText = EncounterJournalTooltipHeaderText {
 }
 0 = <userdata>
 GetBackdropBorderColor = <function> defined @SharedXML/SharedTooltipTemplates.lua:199
 SetBackdropColor = <function> defined @SharedXML/SharedTooltipTemplates.lua:187
 layoutType = "TooltipDefaultLayout"
 clickText = EncounterJournalTooltipClickText {
 }
}
(*temporary) = "BOTTOMLEFT"
(*temporary) = Frame {
 0 = <userdata>
 iconRingHighlight = Texture {
 }
 data = <table> {
 }
 iconRing = Texture {
 }
 icon = Texture {
 }
 text = FontString {
 }
}
(*temporary) = "TOPRIGHT"
(*temporary) = 0
(*temporary) = 0
```

**Comment by jaredbwasserman** [comment]
Fixed in v2.2.4.
